ethclient/gethclient/gethclient.go

```diff
@@ -204,6 +220,46 @@ func (ec *Client) SubscribePendingTransactions(ctx context.Context, ch chan<- co
 	return ec.c.EthSubscribe(ctx, ch, "newPendingTransactions")
 }
 
+// TraceCall lets you trace a given eth_call
+func (ec *Client) TraceCall(ctx context.Context, args ethapi.TransactionArgs, blockNrOrHash rpc.BlockNumberOrHash, config *tracers.TraceCallConfig) (interface{}, error) {
```

Comment:
`ethapi.TransactionArgs` is in an `internal` directory which means users cannot import and use it.

```suggestion
func (ec *Client) TraceCall(ctx context.Context, args ethereum.CallMsg, blockNrOrHash rpc.BlockNumberOrHash, config *tracers.TraceCallConfig) (interface{}, error) {
```